client-sdk/ts-web/rt/test/mrae.test.ts

```diff
@@ -0,0 +1,16 @@
+import * as oasisRT from './../src';
+import * as ed from '@noble/ed25519';
+import * as oasis from '@oasisprotocol/client';
+
+describe('mrae', () => {
+  describe('symmetricKey', () => {
+    it('Should drive symmetric key correctly', async () => {
+      const privateKeyHex = "c07b151fbc1e7a11dff926111188f8d872f62eba0396da97c0a24adb75161750";
```

Comment:
Can you link to where you found this test vector?

Reply:
I just use some random same string at: https://github.com/oasisprotocol/oasis-core/pull/4772
```
 oasis-core/runtime/src/common/crypto/mrae/deoxysii.rs
 oasis-core/go/common/crypto/mrae/deoxysii/asymmetric_test.go
```